cmd/kubeadm/app/cmd/config.go

```diff
@@ -128,26 +129,26 @@ func NewCmdConfigPrintDefault(out io.Writer) *cobra.Command {
 }
 
 func getDefaultAPIObjectBytes(apiObject string) ([]byte, error) {
-	if apiObject == masterConfig {
-
-		internalcfg, err := configutil.ConfigFileAndDefaultsToInternalConfig("", &kubeadmapiv1alpha3.MasterConfiguration{
+	var internalcfg runtime.Object
+	var err error
+	switch apiObject {
+	case masterConfig:
+		internalcfg, err = configutil.ConfigFileAndDefaultsToInternalConfig("", &kubeadmapiv1alpha3.MasterConfiguration{
 			BootstrapTokens: []kubeadmapiv1alpha3.BootstrapToken{sillyToken},
 		})
-		kubeadmutil.CheckErr(err)
-
-		return kubeadmutil.MarshalToYamlForCodecs(internalcfg, kubeadmapiv1alpha3.SchemeGroupVersion, kubeadmscheme.Codecs)
-	}
-	if apiObject == nodeConfig {
-		internalcfg, err := configutil.NodeConfigFileAndDefaultsToInternalConfig("", &kubeadmapiv1alpha3.NodeConfiguration{
+	case nodeConfig:
+		internalcfg, err = configutil.NodeConfigFileAndDefaultsToInternalConfig("", &kubeadmapiv1alpha3.NodeConfiguration{
 			Token: sillyToken.Token.String(),
 			DiscoveryTokenAPIServers:               []string{"kube-apiserver:6443"},
 			DiscoveryTokenUnsafeSkipCAVerification: true,
```

Comment:
We need this for passing validation, but afterwards we should set to false because it is the recommended value

Reply:
added a TODO, please file an issue